FitBitApi/FitBitApi/Modules/Presenter/Model/PresenterModel.swift

```diff
@@ -0,0 +1,26 @@
+//  
+//  PresenterModel.swift
+//  FitBitApi
+//
+//  Created by Wishell on 14.11.2018.
+//  Copyright © 2018 Wishell. All rights reserved.
+//
+
+import Foundation
+
+final class PresenterModel: PresenterModelInput {
+    
+    weak var output: PresenterModelOutput!
+    let repository = Repository(apiClient: APIClient())
```

Comment:
declare as var repository: Repository!

setup in configurator

model.repository = Repository(apiClient: APIClient())